frontend/e2e-tests/dom-to-db-tests/data-entry.e2e.ts

```diff
@@ -15,7 +15,80 @@ import {
+  const recountedPage = new RecountedPage(page);
+  await expect(recountedPage.yes).toBeFocused();
+  await recountedPage.no.check();
+  await expect(recountedPage.no).toBeChecked();
+  await recountedPage.next.click();
+
+  const votersAndVotesPage = new VotersAndVotesPage(page);
+  await expect(votersAndVotesPage.pollCardCount).toBeFocused();
+  const voters: VotersCounts = {
+    poll_card_count: 1000,
+    proxy_certificate_count: 50,
+    voter_card_count: 75,
+    total_admitted_voters_count: 1125,
+  };
+  const votes: VotesCounts = {
+    votes_candidates_count: 1090,
+    blank_votes_count: 20,
+    invalid_votes_count: 15,
+    total_votes_cast_count: 1125,
+  };
+  await votersAndVotesPage.inputVotersCounts(voters);
+  await votersAndVotesPage.inputVotesCounts(votes);
+  await expect(votersAndVotesPage.pollCardCount).toHaveValue(formatNumber(voters.poll_card_count));
+  await votersAndVotesPage.next.click();
+
+  const differencesPage = new DifferencesPage(page);
+  await expect(differencesPage.moreBallotsCount).toBeFocused();
+  await differencesPage.next.click();
+
+  const candidatesListPage_1 = new CandidatesListPage(page, "Lijst 1 - Political Group A");
+  await expect(candidatesListPage_1.getCandidate(0)).toBeFocused();
+
+  await candidatesListPage_1.fillCandidatesAndTotal([837, 253], 1090);
+  const responsePromise = page.waitForResponse(new RegExp("/api/polling_stations/1/data_entries/([12])"));
+  await candidatesListPage_1.next.click();
+
+  const response = await responsePromise;
+  expect(response.status()).toBe(200);
+  expect(response.request().postDataJSON()).toMatchObject(noRecountNoDifferencesRequest);
+  expect(await response.json()).toMatchObject(noErrorsWarningsResponse);
+
+  const checkAndSavePage = new CheckAndSavePage(page);
+  await checkAndSavePage.fieldset.waitFor();
+
+  await expect(checkAndSavePage.summaryText).toContainText(
+    "De aantallen die je hebt ingevoerd in de verschillende stappen spreken elkaar niet tegen. Er zijn geen blokkerende fouten of waarschuwingen.",
+  );
+
+  const expectedSummaryItems = [
+    { text: "Alle optellingen kloppen", iconLabel: "opgeslagen" },
+    { text: "Er zijn geen blokkerende fouten of waarschuwingen", iconLabel: "opgeslagen" },
+    { text: "Je kan de resultaten van dit stembureau opslaan", iconLabel: "opgeslagen" },
+  ];
+  const listItems = checkAndSavePage.allSummaryListItems();
+  const expectedTexts = Array.from(expectedSummaryItems, (item) => item.text);
+  await expect(listItems).toHaveText(expectedTexts);
+
+  for (const expectedItem of expectedSummaryItems) {
+    await expect(checkAndSavePage.summaryListItemIcon(expectedItem.text)).toHaveAccessibleName(expectedItem.iconLabel);
+  }
+
+  await checkAndSavePage.save.click();
+
+  const pollingStationChoicePage = new PollingStationChoicePage(page);
+  await pollingStationChoicePage.fieldsetNextPollingStation.waitFor();
+  await pollingStationChoicePage.dataEntrySuccess.waitFor();
```

Comment:
Commenting here so that it's at least on related code.

After completing the second data entry, the green success banner contains the text: "Geef het papieren proces-verbaal terug aan de coördinator. Een andere invoerder doet straks de tweede invoer."

So we need a different message for when someone completes second data entry. @jorisleker Do you have any suggestions?

And then we need to update both the tests for first and second data entry to check for the full text instead of just the heading ("Je invoer is opgeslagen") as they do now.

Reply:
I've added [a design for this case](https://www.figma.com/design/zZlFr8tYiRyp4I26sh6eqp/Kiesraad---Abacus-optelsoftware?node-id=6996-20519&node-type=frame&t=YE0uHtZoj3WUwn8q-11). Message is very similar to first data entry succes message, we can suffice by just leaving out the last part: 

> 👍 **Je invoer is opgeslagen**
> Geef het papieren proces-verbaal terug aan de coördinator. ~Een andere invoerder doet straks de tweede invoer.~

Reply:
Updated in 7548627acce8921970bb56eca95de4c764732c7d!